api/v1beta1/hosted_controlplane.go

```diff
@@ -210,16 +218,28 @@ type HostedControlPlaneStatus struct {
 	// +kubebuilder:validation:Optional
 	OAuthCallbackURLTemplate string `json:"oauthCallbackURLTemplate,omitempty"`
 
-	// Version is the semantic version of the release applied by
-	// the hosted control plane operator
-	// +kubebuilder:validation:Optional
-	Version string `json:"version,omitempty"`
+	// VersionStatus is the status of the release version applied by the
+	// hosted control plane operator.
+	// +optional
+	VersionStatus *ClusterVersionStatus `json:"versionStatus,omitempty"`
```

Comment:
I assume we want to do this consistently in HostedClusterStatus?

Reply:
HostedControlPlaneStatus had a pre-existing `Version`, so the new property is coming in as `VersionStatus`.  HostedClusterStatus had a pre-existing `Version *ClusterVersionStatus`.  Are you asking me to deprecate `Version` there in favor of a new `VersionStatus` with duplicate information, just to name-match properties in HostedControlPlane?  Personally, HostedControlPlane seems pretty internal,  so I'd rather stick with the nicer `Version` in the more-public HostedCluster, even if that property name diverges from HostedControlPlane's `VersionStatus` property name.  But :shrug:, it's your repository, you can just tell me what you want it to be.

Reply:
I missed that context, makes sense!